app/graphql/types/customers/object.rb

```diff
@@ -32,6 +32,7 @@ class Object < Types::BaseObject
       field :zipcode, String, null: true
 
       field :metadata, [Types::Customers::Metadata::Object], null: true
+      field :tax_rates, [Types::TaxRates::Object], null: true
```

Comment:
Should we expose the `applied_tax_rates`, rather than the `tax_rates`? Otherwise, I'm not sure how the front-end will be able to fetch them (for destroy purpose in particular)

Reply:
Also, maybe we should a request for these attributes somewhere in the specs so that we can validate it serializes well